### PR TITLE
Highest/lowest scoring home team and highest/lowest scoring visitor methods mr

### PR DESCRIPTION
**Pull request by mronauli**
Add methods for highest_scoring_visitor and highest_scoring_home_team for iteration three 
-converting to string name in process

### CODE REVIEW

File: test/game_teams_collection_test.rb [hunk 1]
```diff
@@ -21,6 +23,46 @@ def test_it_can_create_games_from_csv
     assert_equal "LOSS", @game_teams.result
   end
 
+  def test_it_can_store_game_teams_by_id
```
Comment (tylertomlinson) [inline test/game_teams_collection_test.rb:26]: Nice tests

File: test/game_teams_collection_test.rb [hunk 2]
```diff
@@ -21,6 +23,46 @@ def test_it_can_create_games_from_csv
...
+  def test_it_can_get_home_games_only_average
+    assert_equal 2.33, @game_teams_collection.home_games_only_average[26]
+  end
+
+  def test_it_can_get_away_games_only
+    assert_equal 2, @game_teams_collection.away_games_only[26].length
+  end
+
+  def test_it_can_get_away_games_only_average
+    assert_equal 2.0, @game_teams_collection.away_games_only_average[26]
```
Comment (tylertomlinson) [inline test/game_teams_collection_test.rb:47]: NICe

**Review by tylertomlinson — COMMENTED** [review]
Looks good!!!!!!!!!!

File: test/game_teams_collection_test.rb (hunk 2 above)
Comment (mronauli) [reply test/game_teams_collection_test.rb:47]: cool

File: test/game_teams_collection_test.rb [hunk 3]
```diff
@@ -32,19 +32,19 @@ def test_it_can_store_game_teams_by_id
...
   end
 
   def test_it_can_get_home_games_only_average
-    assert_equal 2.33, @game_teams_collection.home_games_only_average["26"]
+    assert_equal 2.33, @game_teams_collection.home_games_only_average[26]
   end
 
   def test_it_can_get_away_games_only
-    assert_equal 2, @game_teams_collection.away_games_only["26"].length
+    assert_equal 2, @game_teams_collection.away_games_only[26].length
```
Comment (Yetidancer) [inline test/game_teams_collection_test.rb:43]: these tests can GIT IT

Comment (mronauli) [reply test/game_teams_collection_test.rb:43]: great!! thanks

**Review by d-atkins — APPROVED** [review]
I see no issues.

**Pull request merged**